content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,154 @@
+  to deploy onto. These can be enforced at the namespace and cluster level with
+  the
+  [PodNodeSelector](/docs/reference/access-authn-authz/admission-controllers/#podnodeselector)
+  admission controller.
+- [PodTolerationRestriction](/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction)
+  is an admission controller that allows administrators to add permitted
+  [tolerations](/docs/concepts/scheduling-eviction/taint-and-toleration/)
+  within a namespace. Pods within that namespace may only utilise those
+  specified in the namespace.
+
+## Secrets
+
+Secrets required for pods should be stored within Kubernetes Secrets as opposed
+to alternatives such as ConfigMap. Secret resources stored within etcd should
+be [encrypted at rest](/docs/tasks/administer-cluster/encrypt-data/).
+
+Pods needing secrets should have these automatically mounted through volumes,
+preferably stored in memory like with the
+[`emptyDir.medium` option](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir).
+This should be done preferentially as compared to providing the pods service
+account RBAC access to secrets. This would allow adding secrets into the pod as
+environment variables or files. However, the environment variable method might
+be more prone to leakage due to crash dumps in logs and the non-confidential
+nature of environment variable in Linux, as opposed to the permission mechanism
+on files.
+
+Service accounts should not be mounted into pods that do not require them. This can be configured by setting
+[`automountServiceAccountToken`](/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server)
+to `false` either within the service account to apply throughout the namespace
+or specifically for a pod. For Kubernetes v1.22 and above, use
+[Bound Service Accounts](/docs/reference/access-authn-authz/service-accounts-admin/#bound-service-account-token-volume)
+for time-bound service account credentials.
+
+## Container & Image scanning
+
+Scanning a container image can prevent critical vulnerabilities from being
+deployed to the cluster alongside the container image. Image scanning should be
+completed before deploying a container image to a cluster and is usually done
+as part of the deployment process in a CI/CD pipeline. The purpose of an image
+scan is to obtain information about possible vulnerabilities and their
+prevention in the container image, such as a
+[Common Vulnerability Scoring System (CVSS)](https://en.wikipedia.org/wiki/Common_Vulnerability_Scoring_System)
+score. If the result of the image scans is combined with the pipeline
+compliance rules, only properly patched container images will end up in
+Production.
+
+The blog post
+"[A Closter Look at NSA/CISA Kubernetes Hardening Guidance](/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/#building-secure-container-images)"
+provides more guidance on how to build secure container images.
```

Comment:
Made a super quick list, but I may have missed some.. 

```suggestion
provides more guidance on how to build secure container images.

## Admission Controllers

The following are a number of admission controllers that could be considered to help increase the security posture of your application, this list includes controllers that may be referenced in other areas of this document:

- `AlwaysPullImages` - Enforces the latest version of a tagged image is used, and the deployer has permissions to use the image
- `DenyServiceExternalIPs` - Prevents usage of `externalIPs` services
- `EventRateLimit` - Rate limts events to the API server
- `ImagePolicyWebhook` - Allows enforcing additional controls for images through webhooks
- `MutatingAdmissionWebhook` - Allows the use of custom controllers through webhooks, these controllers may mutate requests that it reviews
- `NodeRestriction` - Restricts Kubelet permissions based of deployments upon their nodes
- `PodNodeSelector` - Allows controls of node selectors within namespaces and cluster-wide
- `PodSecurity` - Replacement for Pod Security Policy, restricts security contexts of deployed Pods
- `PodTolerationRestriction` - Allows control of pod tolerations permitted for pods within a namespace
- `ResourceQuota` - Enforces resource quotas to prevent over-usage of resources
- `ValidatingAdmissionWebhook` - Allows the use of custom controllers through webhooks, these controllers do not mutate requests that it reviews
```

Reply:
Ok I added the three mentioned by Tim, put hyperlinks and a note.
```suggestion
## Admission Controllers

The following are a number of admission controllers that could be considered to
enhance the security posture of your application. This list includes
controllers that may be referenced in other parts of this document:

- [`AlwaysPullImages`](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages)
  \- Enforces the latest version of a tagged image is used, and the deployer has
    permissions to use the image.
- [`CertificateApproval`](/docs/reference/access-authn-authz/admission-controllers/#certificateapproval)
  \- Performs additional authorization checks to ensure the approving user
    permission to approve certificate request.
- [`CertificateSigning`](/docs/reference/access-authn-authz/admission-controllers/#certificatesigning)
  \- Performs an additional authorization checks to ensure the signing user has
    permission to sign certificate requests.
- [`CertificateSubjectRestriction`](/docs/reference/access-authn-authz/admission-controllers/#certificatesubjectrestriction)
  \- Rejects any request that specifies a 'group' (or 'organization attribute')
    of `system:masters`.
- [`DenyServiceExternalIPs`](/docs/reference/access-authn-authz/admission-controllers/#denyserviceexternalips)
  \- Prevents usage of `externalIPs` services
- [`EventRateLimit`](/docs/reference/access-authn-authz/admission-controllers/#eventratelimit)
  \- Rate limts events to the API server.
- [`ImagePolicyWebhook`](/docs/reference/access-authn-authz/admission-controllers/#imagepolicywebhook)
  \- Allows enforcing additional controls for images through webhooks.
- [`MutatingAdmissionWebhook`](/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook)
  \- Allows the use of custom controllers through webhooks, these controllers
    may mutate requests that it reviews.
- [`NodeRestriction`](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)
  \- Restricts Kubelet permissions based of deployments upon their nodes.
- [`PodNodeSelector`](/docs/reference/access-authn-authz/admission-controllers/#podnodeselector)
  \- Allows controls of node selectors within namespaces and cluster-wide.
- [`PodSecurity`](/docs/reference/access-authn-authz/admission-controllers/#podsecurity)
  \- Replacement for Pod Security Policy, restricts security contexts of
    deployed Pods.
- [`PodTolerationRestriction`](/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction)
  \- Allows control of pod tolerations permitted for pods within a namespace.
- [`ResourceQuota`](/docs/reference/access-authn-authz/admission-controllers/#resourcequota)
  \- Enforces resource quotas to prevent over-usage of resources.
- [`ValidatingAdmissionWebhook`](/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook)
  \- Allows the use of custom controllers through webhooks, these controllers do
    not mutate requests that it reviews.

{{< note >}}
Many of these admission controllers are [enabled by default](/docs/reference/access-authn-authz/admission-controllers/#which-plugins-are-enabled-by-default) by the API server.
{{< /note >}}
```